OTAnalytics/plugin_ui/customtkinter_gui/toplevel_template.py

```diff
@@ -1,45 +1,27 @@
 from abc import ABC, abstractmethod
 from typing import Any
 
-from customtkinter import CTkButton, CTkFrame, CTkToplevel
+from customtkinter import CTkToplevel
 
-from OTAnalytics.application.config import ON_MAC
-from OTAnalytics.plugin_ui.customtkinter_gui.constants import PADX, PADY, tk_events
+from OTAnalytics.plugin_ui.customtkinter_gui.constants import tk_events
 
 
 class ToplevelTemplate(CTkToplevel, ABC):
     def __init__(
         self,
+        title: str,
         initial_position: tuple[int, int],
         **kwargs: Any,
     ) -> None:
         super().__init__(**kwargs)
-        self._get_frame_ok_cancel()
+        self.title(title)
         self._set_ok_cancel_bindings()
         self._set_initial_position(initial_position)
-        self._set_focus()
```

Comment:
Why do we not centrally set the focus to the toplevel?

Reply:
Done :)